src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -49,12 +49,25 @@ public CommandResult execute() {
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            findMatchingPerson(keywords, matchedPersons, person);
+        }
+        return matchedPersons;
+    }
+
+    /**
+     * Retrieves person in the address book whose name contains some of the specified keywords.
+     *
+     * @param keywords for searching
+     * @return list of persons found
```

Comment:
hmm, here you used a @return annotation but you're passing the return through parameter, which is not a very good practice.